src/main/resources/library/attribute-definitions.json

```diff
@@ -165,7 +184,8 @@
     },
     "library:primaryDiseaseSite": {
       "title": "Primary Disease Site",
-      "type": "string"
+      "type": "string",
+      "typeahead": "populate"
```

Comment:
i'm assuming we might have different types of typeahead in the future. so i think it would be better to have this be stating what kinds of type ahead apply instead of a negative